### ISSUE TITLE
Stack version recently changed?

### ISSUE THREAD

**Issue by sharris-pjr** [jdebloat/jdebloat#3]
Howdy  #all,
Has the stack version recently changed from the documented 1.9.3?
When pulling down the latest jdebloat/jshrink/jreduce/jinline, we get the error below unless we upgrade stack to 2.7.4.

Thank you!

./jdebloat.py setup
Downloading lts-17.11 build plan ...
RedownloadInvalidResponse Request {
  host                 = "raw.githubusercontent.com"
  port                 = 443
  secure               = True
  requestHeaders       = []
  path                 = "/fpco/lts-haskell/master//lts-17.11.yaml"
  queryString          = ""
  method               = "GET"
  proxy                = Nothing
  rawBody              = False
  redirectCount        = 10
  responseTimeout      = ResponseTimeoutDefault
  requestVersion       = HTTP/1.1
}
 "/home/scott/.stack/build-plan/lts-17.11.yaml" (Response {responseStatus = Status {statusCode = 404, statusMessage = "Not Found"}, responseVersion = HTTP/1.1, responseHeaders = [("Connection","keep-alive"),("Content-Length","14"),("Content-Security-Policy","default-src 'none'; style-src 'unsafe-inline'; sandbox"),("Strict-Transport-Security","max-age=31536000"),("X-Content-Type-Options","nosniff"),("X-Frame-Options","deny"),("X-XSS-Protection","1; mode=block"),("Content-Type","text/plain; charset=utf-8"),("X-GitHub-Request-Id","BDB2:56BD:20CB8:2EF7B:617893AE"),("Accept-Ranges","bytes"),("Date","Tue, 26 Oct 2021 23:47:58 GMT"),("Via","1.1 varnish"),("X-Served-By","cache-bur17523-BUR"),("X-Cache","MISS"),("X-Cache-Hits","0"),("X-Timer","S1635292079.620972,VS0,VE186"),("Vary","Authorization,Accept-Encoding,Origin"),("Access-Control-Allow-Origin","*"),("X-Fastly-Request-ID","ead8598e4c913d9542bf27168977f6d390b7ab2f"),("Expires","Tue, 26 Oct 2021 23:52:58 GMT"),("Source-Age","0")], responseBody = (), responseCookieJar = CJ {expose = []}, responseClose' = ResponseClose})
JInline: creating DB...
cd external/soot; mvn clean compile assembly:single

**Comment by akshayutture** [comment]
Yes. The stack version needs to be updated to install the latest version of JReduce. 

Just run 'stack upgrade' to get the latest version of stack. That should work.
